CUSTOMERS_ANALYSIS/readme.md

```diff
@@ -0,0 +1,51 @@
+# Customer Analysis
+
+<ins>**Overview**</ins>
+
+The primary purpose of a business is to make a profit, which is crucial for its survival and growth.
+Data collection and analysis can enable a company understand its customer base and
+market trends, ultimately leading to more targeted marketing strategies,
+efficient resource allocation and enhanced profitability.
+
+<ins>**Data Structure**</ins>
+
+The data follows Medallion architecture, see definition by **[Databricks](https://www.databricks.com/glossary/medallion-architecture)**
+
+The bronze(raw) table consists of two tables(cvs file):customers and orders table
+
+**Table 1: Customers Information**
+
+- id(int):is a unique identifier for each customer,
+- firstname(text):stores the customer's given first name
+- lastname(text): stores the customer's given last name
+- email(text): holds the customer's email address
+- city(text): indicates the customer's city of residence
+- birth_date(text): records the customer's date of birth
+
+**Table 2: Orders Information**
+
+- orderID(text): serves as a unique identifier for each order placed
+- customer_id(int): links to the ID of the customer who placed the order
+- order_amount(int): the value of the order
+- currency(text):the currency in which the order was made
+- full_country_name(text): the country of the order's origin
+- purchase_date(varchar): indicates date of order
+- product_category(text):the category the order belongs to
+
+<ins>**Code Desription**</ins>
+
+The script starts with running codes for cleaning the bronze customer table and orders table into an silver table for the following analysis following analysis:
```

Comment:
"The script starts by cleaning.." (codes should anyways always be singular, i.e. "running code").